content/articles/how-to-create-a-machine-learning-app-using-turicreate/index.md

```diff
@@ -0,0 +1,413 @@
+
+- To use the logistic regression algorithm use the foloowing command:
+
+```python
+logistic_model = tc.logistic_classifier.create(train_data,target='class',features=feature_names)
+```
+
+Output:
+
+```bash
+PROGRESS: Creating validation set data from 75% of train_data.
+
+
+Logistic regression:
+--------------------------------------------------------
+Number of examples: 350
+Number of classes: 2
+Number of feature columns: 16
+Number of unpacked features: 16
+Number of coefficients: 17
+Starting Newton Method
+--------------------------------------------------------
++-----------+----------+--------------+-------------------+---------------------+
+| Iteration | Passes   | Elapsed Time | Training Accuracy | Validation Accuracy |
++-----------+----------+--------------+-------------------+---------------------+
+| 1         | 2        | 0.014390    | 0.927183         | 0.897374            |
+| 2         | 3        | 0.016456    | 0.931734         | 0.843725            |
+| 3         | 4        | 0.018732    | 0.949024         | 0.843725            |
+| 4         | 5        | 0.020935    | 0.957320         | 0.843725            |
+| 5         | 6        | 0.023619    | 0.960135         | 0.843725            |
+| 7         | 8        | 0.026907    | 0.969716         | 0.843725            |
++-----------+----------+--------------+-------------------+---------------------+
+SUCCESS: Optimal solution found.
+```
+
+TuriCreate will be able to use both the training accuracy and Validation accuracy through the iteration. Here we iterate 7 times, and the Training accuracy and Validation accuracy after 7 iterations will be 0.965517 0.842105, respectively.
+
+### Getting the model summary
+
+```python
+logistic_model.summary()
+```
+
+Output:
+
+```bash
+Class                          : LogisticClassifier
+
+Schema
+------
+Number of coefficients: 17
+Number of examples: 350
+Number of classes: 2
+Number of feature columns: 16
+Number of unpacked features: 16
+
+Hyperparameters
+---------------
+L1 penalty                     : 0.0
+L2 penalty                     : 0.01
+
+Training Summary
+----------------
+Solver: newton
+Solver iterations: 7
+Solver status                  : SUCCESS: Optimal solution found.
+Training time (sec)            : 0.0333
+
+Settings
+--------
+Log-likelihood: 42.9568
+
+Highest Positive Coefficients
+-----------------------------
+Gender[Female]                 : 5.1663
+Irritability[Yes]              : 3.5232
+Itching[No]                    : 3.4068
+Polyuria[Yes]                  : 3.2939
+Genital thrush[Yes]            : 2.4532
+
+Lowest Negative Coefficients
+----------------------------
+Polydipsia[No]                 : -6.5078
+Alopecia[No]                   : -0.8826
+weakness[No]                   : -0.283
+Age                            : -0.0729
+```
+
+### Model Evaluation
+
+This is assessing our model to find out how well it learned. We do this by using the test_data
```

Comment:
Whenever you refer to variables used in code, use \`\` to wrap the variable. Example: \`test_data\`